charts/script-exporter/templates/deployment.yaml

```diff
@@ -30,6 +30,8 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          {{- toYaml .Values.env | nindent 10 }}
```

Comment:
```suggestion
          {{- with .Values.cluster.env }}
          env:
            {{- toYaml . | nindent 12 }}
          {{- end }}
```

Reply:
Is there any reason why you are suggesting to put into `.Values.cluster.en` ? instead of `.Values.env` ?

Reply:
I added the env to the container and I don't see `.Values.cluster.env` defined in the `values.yaml`
```
## Specify additional environment variables for the script_exporter container.
##
env: []
  # - name: MY_ENV_VAR
  #   value: MY_ENV_VALUE
```

Reply:
Sry my fault, I just copied it from one of my other Helm charts and forgot to adjust it. The suggestion should be fixed now.